grakn-test/src/test/java/ai/grakn/generator/ConceptIds.java

```diff
@@ -22,7 +22,7 @@
 import ai.grakn.concept.ConceptId;
 
 /**
- * Generator that generates totally concept IDs
+ * Generator that generates totally random concept IDs
  */
 public class ConceptIds extends AbstractGenerator<ConceptId> {
```

Comment:
I agree on the monolith thing and I'll move them back to `grakn-test-tools`.

Here's the layout:
- Generators for `grakn-graph` will go in `grakn-test-tools`
- Property tests for `grakn-graph` will go in `grakn-test`
- Generators and property tests for `grakn-graql` will go in `grakn-graql`

I'm not super-happy with this - the layout is scattered and inconsistent. We'll also have issues if we ever need to write say, tests for `grakn-engine` that generate queries. On the other hand I have no better ideas... except Gradle of course 👍